zebra-chain/src/block/test_vectors.rs

```diff
@@ -0,0 +1,97 @@
+// Copied from librustzcash
+// From mainnet block 415000.
+pub const HEADER_MAINNET_415000: [u8; 1487] = [
```

Comment:
:+1: is this used in a test yet? it looks like it has a leading 4, so i'm curious whether it works correctly with the (de)serialize impls.

Reply:
This is not currently used yet, I stole it from librustzcash, I'll make a test and see!

Reply:
Updated!